--- unittest2-1.0.1/setup.py.orig	2015-04-12 23:48:07.788577281 +0200
+++ unittest2-1.0.1/setup.py	2015-04-12 23:48:36.288543928 +0200
@@ -57,7 +57,7 @@
 # Both install and setup requires - because we read VERSION from within the
 # package, and the package also exports all the APIs.
 # six for compat helpers
-REQUIRES = ['argparse', 'six>=1.4', 'traceback2'],
+REQUIRES = ['six>=1.4', 'traceback2'],
 
 params = dict(
     name=NAME,
